CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
         public string? Position { get; set; }
```

Comment:
The new `Salary` property is inserted between the `ForeignKey` attribute and its intended target property `CompanyId`, breaking the attribute association and possibly causing data annotation issues.

```suggestion
public Guid CompanyId { get; set; }
```